src/microcosm.js

```diff
@@ -329,10 +259,10 @@ class Microcosm extends Emitter implements Domain {
    * repo.push(createPlanet, { name: 'Merkur' })
    * ```
    */
-  push(command: Command, ...params: *): Action {
+  push(command: any, ...params: *): Action {
```

Comment:
Flow still doesn't like that tagged actions are functions with properties that stringify to a unique id. This is a workaround.